lib/components/map/connected-transit-vehicle-overlay.js

```diff
@@ -138,7 +138,8 @@ const mapStateToProps = (state) => {
           ? '#' + route.color
           : route?.color || '#5A5A5A'
       // Try to populate this attribute, which is required for the vehicle popup to appear.
-      vehicle.routeShortName = vehicle.routeShortName || route?.shortName
+      vehicle.routeShortName =
+        vehicle.routeShortName || route?.shortName || true
```

Comment:
How is this not throwing a type error

Reply:
And what happens when the vehicle popup tries to render the name?

Reply:
not in a typescript file...

Reply:
Yeah it's not great but I think it's the cleanest way to resolve this without re-writing the entire overlay

Reply:
#1121 improves this